# TerimaUdp.msg
# Message untuk data yang diterima dari UDP

float32 posisi_x_buffer
float32 posisi_y_buffer
float32 sudut_w_buffer
float32 kecepatan_robotx
float32 kecepatan_roboty
float32 kecepatan_robotw
float32 sudut_servo_act
uint8 tombol
float32 vx_global
float32 vy_global
float32 vw_global
